spec/plsql/variable_spec.rb

```diff
@@ -99,6 +102,12 @@
       expect(plsql.test_package.nchar_variable).to eq('abc' + ' '*7)
     end
 
+    it 'can use package variables with spaces in size' do
+      expect(plsql.test_package.varchar_with_spaces1).to eq('a string')
+      expect(plsql.test_package.varchar_with_spaces2).to eq('a string')
```

Comment:
Multiple expectations in same example is in general a bit problematic (if 1st fails one fails, the rest won't b executed). I'm wondering if we can apply either RSpec [aggregating_faulures](https://relishapp.com/rspec/rspec-core/docs/expectation-framework-integration/aggregating-failures) or some of the ways for parameterised examples